files/ko/web/html/element/textarea/index.html

```diff
@@ -97,8 +97,6 @@ <h2 id="DOM_인터페이스">DOM 인터페이스</h2>
 
 <p>이 요소는 <code><a href="/en/DOM/HTMLTextAreaElement" title="en/DOM/textarea">HTMLTextAreaElement</a></code> 인터페이스를 구현합니다.</p>
 
-<p>{{ h1_gecko_minversion("Resizable textareas", "2.0") }}</p>
-
 <p>{{ gecko("2.0") }} introduced support for resizable textareas. This is controlled by the {{ cssxref("resize") }} CSS property. Resizing of textareas is enabled by default, but you can explicitly disable it by including the following CSS:</p>
```

Comment:
```suggestion
<p>Gecko 2.0 introduced support for resizable textareas. This is controlled by the <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/resize">resize (en-US)</a> CSS property. Resizing of textareas is enabled by default, but you can explicitly disable it by including the following CSS:</p>
```
> KumaScript macros are being phased out, to help the Yari dev team, here is a list of macros that will disappear from your files.

Could this be the case with macros like this?